truss-chains/truss_chains/stub.py

```diff
@@ -81,17 +82,20 @@ def _client_sync(self) -> httpx.Client:
         assert self._cached_sync_client is not None
         return self._cached_sync_client[0]
 
-    async def _client_async(self) -> httpx.AsyncClient:
+    async def _client_async(self) -> aiohttp.ClientSession:
```

Comment:
I think the typing of everything in the BasetenSession class definition need to change. Additionally, the sync client created above also needs to change. We should ideally try to drop the httpx import and httpx limit definiteion above.

Reply:
what would you propose changing sync to? `requests`? I don't think there's a problem with using httpx for sync, this bug is only apparent in async.

Reply:
okay. that's fine then. Maybe worth adding a comment

Reply:
I'd like to drop the sync path anyway, but there is an unresolved question around that: https://basetenlabs.slack.com/archives/C06RAC0JT5J/p1727900427264919